src/features/devices/hooks/should-be-managed-by--release.ts

```diff
@@ -0,0 +1,34 @@
+import { hooks } from '@balena/pinejs';
```

Comment:
This hook also looks like it should be in the feature that would create the should_be_managed_by__release relationship

Reply:
Need some guidance about what this means (in terms of a practical resolution) ...fyi ... @xginn8 (thanks)